src/Microsoft.ML.DataView/VectorType.cs

```diff
@@ -21,6 +22,15 @@ public sealed class VectorDataViewType : StructuredDataViewType
         /// As with <see cref="Size"/>, a zero value indicates that the vector type is considered to have
         /// unknown length along that dimension.
         /// </summary>
+        /// <remarks>
+        /// In the case where this is a multi-dimensional type, that is, a situation where <see cref="Dimensions"/>
+        /// has length greater than one, since <see cref="VBuffer{T}"/> itself is a single dimensional sstructure,
+        /// we must clarify what we mean. The indices represent a "flattened" view of the coordinates implicit in the
+        /// dimensions. We consider that that last dimension is the most "minor" index. So, if you hypothetically had
+        /// dimensions of <c>{ 5, 2 }</c>, then the <see cref="VBuffer{T}"/> values would be all of length <c>10</c>,
+        /// and the flattened indices in the  of <c>0, 1, 2, 3, 4, ...</c> would correspond to "coordinates" of
+        /// <c>(0, 0), (0, 1), (1, 0), (1, 1), (2, 0), ...</c>, respectively.
+        /// </remarks>
```

Comment:
(Just thinking out loud) Would it help to say for 2 dimensions, this is also called "Row Major" order?  Or as ndarray calls it: order = `C`

Reply:
Yes, I buy that -- it's very specific to two dimensions, but it at least gets people in the right ballpark immediately, I guess.